code/datums/ruins.dm

```diff
@@ -6,9 +6,6 @@
 	How is there a wooden container filled with 18th century coinage in the middle of a lavawracked hellscape? \
 	It is clearly a mystery."
 
-	var/unpickable = FALSE 	 //If TRUE these won't be placed automatically (can still be forced or loaded with another ruin)
-	var/always_place = FALSE //Will skip the whole weighting process and just plop this down, ideally you want the ruins of this kind to have no cost.
-	var/placement_weight = 1 //How often should this ruin appear
```

Comment:
if its actually working, I would like to keep this, having some granularity over more loot heavy ruins will eventually be nice

Reply:
I was just thinking similar, no, it didn't work in this, but it would be easy to reimplement honestly. I'll add that to the PR

Reply:
bitchin

Reply:
Added!